test/testcanvas.go

```diff
@@ -160,8 +160,19 @@ func (c *testCanvas) Resize(size fyne.Size) {
 		return
 	}
 
+	// Ensure testcanvas mimics real canvas.Resize behavior
 	for _, overlay := range overlays.List() {
-		overlay.Resize(size)
+		type popupWidget interface {
+			fyne.CanvasObject
+			ShowAtPosition(fyne.Position)
+		}
+		if p, ok := overlay.(popupWidget); ok {
+			// TODO: remove this when #707 is being addressed.
```

Comment:
I know this is copied from GLFW but I wonder, is this impacted/fixed by the PR?

Reply:
The latest tests I have added to `popup_test.go` give different results (wrong ones) compared with real implementation.

Reply:
Yes I see why the code is added - I was remarking about the bugfix mentioned in the code you coped.
In other areas of the PR you suspected it might be resolved and have removed some of the mentions.

Reply:
Oh I see.  The code I removed isn't needed for the test as it already guarantee that behavior (it maybe should have been removed before ??).  I found # 707 a bit confusing, it seems to be a design proposal, so don't know what decision was taken around it

Reply:
@andydotxyz  Based on the Slack discussion, is it ok to keep this addition to `testcanvas.go`?

Reply:
Yes I think so